### PR TITLE
Hot fix/video player

### PR DESCRIPTION
**Pull request by sherlockvn**
### Improve/ fix:
- Video player can't be played when first time open
- Remove safearea in video player
- Improve the position and size of Seek bar
- Improve the position of back button

### Demo web:
https://github.com/linagora/twake-on-matrix/assets/43041967/66343701-87aa-4b2e-a0ff-740b9999be20

### Demo mobile:
https://photos.app.goo.gl/QSYCb9V9fUaDFUjC9

### CODE REVIEW

File: lib/widgets/video_viewer_dialog.dart [hunk 1]
```diff
@@ -21,56 +20,43 @@ class VideoViewerDialog extends StatefulWidget {
 }
 
 class _VideoViewerDialogState extends State<VideoViewerDialog> {
-  Player? videoPlayer;
-  VideoController? videoController;
+  Player videoPlayer = Player();
+  late final VideoController videoController = VideoController(videoPlayer);
```
Comment (drminh2807) [inline lib/widgets/video_viewer_dialog.dart:24]: why need `late`

File: lib/widgets/video_viewer_dialog.dart [hunk 2]
```diff
@@ -84,22 +70,17 @@ class _VideoViewerDialogState extends State<VideoViewerDialog> {
...
-                  controller: videoController!,
-                ),
-              ),
-            )
-          : const SizedBox(),
+      child: Container(
+        decoration: const BoxDecoration(
+          color: MessageContentStyle.backgroundColorVideo,
+        ),
+        child: Video(
```
Comment (drminh2807) [inline lib/widgets/video_viewer_dialog.dart:77]: removed safe area ?
please demo on iphone pro max

Comment (drminh2807) [reply lib/widgets/video_viewer_dialog.dart:77]: please demo on media chat details page also

**Comment by hoangdat** [comment]
- add primary color for `seekBar`
- hover to player to show controller buttons in web

File: lib/widgets/video_viewer_dialog.dart (hunk 1 above)
Comment (sherlockvn) [reply lib/widgets/video_viewer_dialog.dart]: remove it

File: lib/widgets/video_viewer_dialog.dart (hunk 2 above)
Comment (sherlockvn) [reply lib/widgets/video_viewer_dialog.dart]: okay, you can see it in the link I post above

**Comment by sherlockvn** [comment]
> * add primary color for `seekBar`
> * hover to player to show controller buttons in web


https://github.com/linagora/twake-on-matrix/assets/43041967/ca9e201a-79c7-47ce-9388-bb0b673409df

File: lib/widgets/video_viewer_dialog.dart (hunk 2 above)
Comment (sherlockvn) [reply lib/widgets/video_viewer_dialog.dart]: https://github.com/linagora/twake-on-matrix/assets/43041967/6da4a66e-7326-4993-9a2b-61e34868fcf5

Comment (drminh2807) [reply lib/widgets/video_viewer_dialog.dart]: @sherlockvn  can't see fullscreen player in media page

File: lib/pages/chat/events/sending_video_widget.dart [hunk 3]
```diff
@@ -92,8 +92,9 @@ class SendingVideoWidget extends StatelessWidget {
     await showDialog(
       context: context,
       useRootNavigator: false,
+      useSafeArea: false,
       builder: (context) {
-        return VideoViewerDialog(path: matrixFile.filePath!);
+        return VideoViewerMobileTheme(path: matrixFile.filePath!);
```
Comment (drminh2807) [inline lib/pages/chat/events/sending_video_widget.dart:97]: web also need see video player when sending, right ?

Comment (sherlockvn) [reply lib/pages/chat/events/sending_video_widget.dart]: done

**Review by hoangdat — APPROVED** [review]

**Review by drminh2807 — APPROVED** [review]

**Review by nqhhdev — APPROVED** [review]

**Pull request merged**